tests/epstats/server/test_api_evaluate.py

```diff
@@ -1,16 +1,16 @@
 import pandas as pd
 from fastapi.testclient import TestClient
 
-from epstats.toolkit.testing import (
```

Comment:
It should all work without the need to start all imports with `src`. Running of `make install-dev` should install the package with `pip install -e .` to the python env. So we do not need to start imports with `src`. Repeats many times in changed files.

Reply:
Ahh, ok, I got it. This `src` is just for files with tests. It's ok then and thanks for fixing it!